src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         {{- printf "%s: %s" $key (tpl $value $ | quote) | nindent 8 }}
         {{- end }}
     spec:
+      securityContext:
+      {{- toYaml .Values.podSecurityContext | nindent 8 }}
```

Comment:
I don't have a strong opinion here - I've also seen charts exposed `containerSecurityContext` (for the container level), and have `securityContext` be for the pod itself. Either is OK with me, just went with the minimal changes option initially.